packages/web3/src/utils/group.ts

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2018 - 2022 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { TOTAL_NUMBER_OF_GROUPS } from '../constants'
+import djb2 from './djb2'
+
+export function groupFromBytes(bytes: Uint8Array): number {
```

Comment:
I think this one should be a private function in `address`, as it's only used for asset addresses. The hints for contracts are different.

We could also rename it to be more informative: `groupFromBytesForAssetAddress()`

Reply:
Fixed, thanks :+1: